scripts/lib/install_kafka.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
```

Comment:
Library scripts should not have #!/bin/bash at the top https://google.github.io/styleguide/shell.xml?showone=File_Extensions#File_Extensions

Reply:
Fixed